### PR TITLE
TIAF: Native test opt-in phase 3

### PR DESCRIPTION
**Pull request by jonawals**
## What does this PR do?

This PR opts in the phase 3 native tests for TIAF.

## How was this PR tested?

This PR will be tested in AR.

### CODE REVIEW

File: Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt [hunk 1]
```diff
@@ -44,6 +44,7 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
     )
     ly_add_googletest(
         NAME Gem::AtomViewportDisplayInfo.Tests
+
```
Comment (lumberyard-employee-dm) [inline Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt:47]: Nothing was changed in this file.

**Review by lumberyard-employee-dm — CHANGES_REQUESTED** [review]

File: Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt (hunk 1 above)
Comment (jonawals) [reply Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt]: Fixed, unblock the PR ASAP please.

File: Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt [hunk 2]
```diff
@@ -44,7 +44,7 @@ if(PAL_TRAIT_BUILD_TESTS_SUPPORTED)
     )
     ly_add_googletest(
         NAME Gem::AtomViewportDisplayInfo.Tests
-        
+        LABESL REQUIRES_tiaf
```
Comment (lumberyard-employee-dm) [inline Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt:47]: There is a typo in `LABEL` here
```suggestion
        LABEL REQUIRES_tiaf
```

**Review by lumberyard-employee-dm — CHANGES_REQUESTED** [review]

File: Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt (hunk 1 above)
Comment (lumberyard-employee-dm) [reply Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/CMakeLists.txt]: This is not fixed.

**Review by lumberyard-employee-dm — APPROVED** [review]

**Pull request merged**